EventsExpress/ClientApp/src/containers/inventory-item.js

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import OwnerSeeItem from '../components/inventory/ownerSeeItem';
+import OwnerEditItemForm from '../components/inventory/ownerEditItem';
+import VisitorSeeItem from '../components/inventory/visitorSeeItem';
+import VisitorEditItemForm from '../components/inventory/visitorTakeItem';
+import { get_inventories_by_event_id }  from '../actions/inventory-list';
+import { delete_users_inventory, edit_users_inventory }  from '../actions/usersInventories';
+import { delete_item, edit_item, add_item, want_to_take } from '../actions/inventar';
+
+class InventoryItemWrapper extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            showAlreadyGetDetailed: false
+        };
+
+        this.onAlreadyGet = this.onAlreadyGet.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+        this.onCancel = this.onCancel.bind(this);
+        this.onWillTake = this.onWillTake.bind(this);
+    }
+
+    onAlreadyGet = () => {
+        this.setState(state => ({
+            showAlreadyGetDetailed: !state.showAlreadyGetDetailed
+        }));
+    }
+
+    
+    deleteItemFromList = inventar => {
+        this.props.delete_item(inventar.id, this.props.eventId);
+    }
+
+    markItemAsEdit = inventar => {
+        let updateList = this.props.inventories.items;
+        updateList.find(e => e.id === inventar.id).isEdit = true;
+
+        this.props.get_inventories(updateList);
+        this.props.changeDisableEdit(true);
+    }
+
+    markItemAsWillTake = inventar => {
+        let updateList = this.props.inventories.items;
+        updateList.find(e => e.id === inventar.id).isWillTake = true;
```

Comment:
You are still tracking it inside redux. Why not have a state here? This component corresponds to a single row and can track state of a single row inside of it.